inspektor/check.py

```diff
@@ -245,9 +234,9 @@ def get_parser(self, prog_name):
         parser.add_argument('gh_id', type=int,
                             help='GitHub Pull Request ID')
         parser.add_argument('-p', '--parent-project', type=str,
-                            help=('Parent project name of the current repository.'
-                                  ' Default: %(default)s'),
-                            default='autotest')
+                            help=('Github organization. '
+                                  'Default: %(default)s'),
+                            default='avocado-framework')
```

Comment:
Well it's sad to remove the `patchwork` checker, it might be useful (even though it's not actively maintained by us). Anyway please fix the `_fetch_from_github` docstring which still talks about patchwork and also if you really want to move exclusively to github, I'd recommend using github features like `git fetch` and such to avoid the need to set `github organization`. It could be `git` only as we are talking only about github checker.